sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -3013,7 +3014,23 @@ Accept wildcard characters: False
 When set to `True`, forces a sync of **Content Security Policy** entries for SharePoint framework component in the tenant application catalog.
 New entries will be added to the configuration, if not already present, based on the `cdnBasedPath` property under a solution's `.config/write-manifests.json` if present.
 The sync may take up to 24 hours to complete.
-In multi-geo environments, **Content Security Policy** entries are unique to each geo.
+In multi-geo environments, **Content Security Policy** configuration is unique to each geo.
+
+```yaml
+Type: Boolean
+Parameter Sets: (All)
+Applicable: SharePoint Online
+Required: False
+Position: Named
+Default value: False
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -EnforceContentSecurityPolicyConfiguration
+
+When set to `True`, opts-in to enforcement of the current **Content Security Policy** configuration.
```

Comment:
I would change with: "When set to 'True' Content Security Policy violations will be enforced.

1. When set to 'False' violations are not enforced; administrators will find all the Content Security Policy Violations in Microsoft Pureview Portal."

Reply:
Should we mention the Purview portal before that is ready? We could probably update the docs here with more specific info after the audit events are exposed.